cgmanifest.json

```diff
@@ -13138,6 +13128,16 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "moby-compose",
```

Comment:
I am to blame for this :(

Reply:
Nah, I'm blaming the checks that didn't enforce the correct ordering! I need to file a bug in ADO for that...